table/tables/tables.go

```diff
@@ -437,7 +437,7 @@ func (t *Table) AddRecord(ctx context.Context, r []interface{}) (recordID int64,
 		var value interface{}
 		key := t.RecordKey(recordID, col)
 
-		if col.State == model.StateWriteOnly || col.State == model.StateReorganization {
+		if col.State == model.StateWriteOnly || col.State == model.StateWriteReorganization {
```

Comment:
Any need to extract `v.State == model.StateDeleteOnly || v.State == model.StateDeleteReorganization` check to a function and add some comment?